Packs/TaniumThreatResponse/Integrations/TaniumThreatResponseV2/TaniumThreatResponseV2.py

```diff
@@ -937,10 +989,13 @@ def get_alerts(client, data_args) -> Tuple[str, dict, Union[list, dict]]:
                            limit=limit,
                            offset=offset, state=state.lower() if state else None)
 
-    raw_response = client.do_request('GET', '/plugin/products/detect3/api/v1/alerts/', params=params)
+    raw_response = client.do_request('GET', '/plugin/products/'
+                                            f'{client.get_threat_response_endpoint()}'
+                                            f'/api/v1/alerts/', params=params)
 
     alerts = []
-    for item in raw_response:
+    raw_response_data = raw_response.get("data") if client.api_version == '4.x' else raw_response
```

Comment:
Is `data` present in pre-4.x? Can't you use `normalize_api_response()`?

Reply:
no its not in pre-4.x, that's one of the major changes, only in very specific cases which also already have the new api endpoint

Reply:
Can't you use `normalize_api_response` instead?